flink-runtime/src/main/java/org/apache/flink/runtime/leaderretrieval/LeaderRetrievalEventHandler.java

```diff
@@ -36,6 +36,8 @@
 	 *
 	 * @param leaderInformation the new leader information to notify {@link LeaderRetrievalService}. It could be
 	 * {@link LeaderInformation#empty()} if the leader address does not exist in the external storage.
+	 * Duplicated leader change events could happen, so the implementation should check whether the passed leader
+	 * information is truly changed with last stored leader information.
```

Comment:
I think this should not go below the `@param` description but the general description.